dwave_networkx/algorithms/elimination_ordering.py

```diff
@@ -125,7 +134,7 @@ def minor_min_width(G):
     """
     # we need only deal with the adjacency structure of G. We will also
     # be manipulating it directly so let's go ahead and make a new one
-    adj = {v: set(G[v]) for v in G}
+    adj = {v: set(u for u in G[v] if u != v) for v in G}
```

Comment:
Just wondering, seeing as this is done 6 times in this module, why not define it as an internal func?
```
def _get_adj(G):
     """<condensed version of> #we need only deal with the adjacency structure of G. We will also
         # be manipulating it directly so let's go ahead and make a new one
    """
    return {v: set(u for u in G[v] if u != v) for v in G}
```

Reply:
In this case I think one line functions are better duplicated, especially when they are not doing complicated logic. Especially since we would probably need to keep the comment string in place - I cannot think of a nice way to name that's more explicit than the line itself.